src/my_module.c

```diff
@@ -3,11 +3,7 @@
  * Copyright 2024 Aleksandra Smolniakova.
  */
 
-#include <linux/fs.h>
-#include <linux/init.h>
 #include <linux/module.h>
```

Comment:
minor: only this header is actually needed

Reply:
Indeed!